get/build.go

```diff
@@ -84,7 +88,7 @@ func printBuild(builds []apps.Build, get *Cmd, out io.Writer, header bool) error
 }
 
 func pullImage(ctx context.Context, apiClient *api.Client, build *apps.Build) error {
-	cli, err := client.NewClientWithOpts(client.FromEnv)
+	cli, err := client.NewClientWithOpts(client.WithVersion(dockerAPIVersion))
```

Comment:
something which just came to my mind:
As we are leaving the `client.FromEnv` out now, we remove the possibility to overwrite the version or any other settings via environment variables.  Maybe we can use both options?

```go
cli, err := client.NewClientWithOpts(client.WithVersion(dockerAPIVersion), client.FromEnv)
```

This might set a default for the version to use which can then still be overwritten by env variables. Could you try that?